app/index.js

```diff
@@ -7,11 +7,6 @@ import Root from './components/Root'
 import './styles/tippy.compiled.global.css'
 import './styles/main.global.scss'
 
-// eslint-disable-next-line
-window.eval = global.eval = function() {
-  throw new Error('Sorry, this app does not support window.eval().')
-}
-
```

Comment:
This was causing an error originating from neon-js when trying to encrypt a key 👎 so I am eliminating this block for now